scripts/debuild.sh

```diff
@@ -3,7 +3,9 @@
 # on travis in /home/travis/build/fossasia/meilix
 
 #rm meilix-default-settings_*                                    #removes the older meilix-default-settings packages if exist
-cd meilix-default-settings                                      #cd into the metapackage directory
+cd meilix-default-settings                                 #cd into the metapackage directory 
+cd ..
+sudo rm melix-default-settings                                  #again trying to remove the melix-default settings package if exist from root user
```

Comment:
You remove it, then you debuild it? I don't get it.

Reply:
see line 7

Reply:
@tabesin Could you pls guide me towards this issue??